var/spack/repos/builtin/packages/cudnn/package.py

```diff
@@ -190,6 +190,13 @@ def url_for_version(self, version):
 
         return url.format(directory, cuda, ver)
 
+    def setup_run_environment(self, env):
+        if 'target=ppc64le: platform=linux' in self.spec:
+            env.set('cuDNN_ROOT', os.path.join(
```

Comment:
@bvanessen to make sure, this needs to set `cuDNN_ROOT` and `CUDNN_ROOT`?

Reply:
I changed it to only set one.